crmsh/lock.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (C) 2020 Xin Liang <XLiang@suse.com>
+# See COPYING for license information.
+
+
+import re
+import time
+from contextlib import contextmanager
+
+from . import utils
+from . import config
+from .msg import common_warn
+
+
+class SSHError(Exception):
+    """
+    Custom exception for ssh error
+    """
+    pass
```

Comment:
We don't need this `pass`. Just remove it, the exception will work fine

Reply:
Changed